### PR TITLE
Draft of docs for storage migration

### PR DESCRIPTION
**Pull request by kyleprice**

### CODE REVIEW

File: src/markdown-pages/install-with-kurl/migrating-csi.md [hunk 1]
```diff
@@ -180,6 +180,43 @@ To migrate to a new CSI provisioner in a kURL cluster:
 
    During the cluster upgrade, the kURL installer detects that the CSI add-on has changed. kURL automatically begins the process of migrating data from the current CSI provisioner to the provisioner in the updated specification. For more information about the data migration process, see [About Changing the CSI Add-on](#about-changing-the-csi-add-on) above.
 
+## Automated Local to Distributed Storage Migrations
+
+kURL is able to migrate clusters from local (Non-HA) storage to distributed (HA) storage when it reaches n number of nodes. After a node greater than or equal to n join the cluster, the join script will prompt to perform the migration. This migration is also available as a task within task.sh.  Please note the following requirements when choosing to have customers on both local and distributed storage.
```
Comment (rrpolanco) [inline src/markdown-pages/install-with-kurl/migrating-csi.md:185]: >kURL is able to migrate clusters from local (Non-HA) storage to distributed (HA) storage when it reaches n number of nodes

We talked about N=3 being the minimum value and so we should explicitly state this.

Comment (rrpolanco) [inline src/markdown-pages/install-with-kurl/migrating-csi.md:185]:
```suggestion
kURL is able to migrate clusters from local (Non-HA) storage to distributed (HA) storage when the cluster expands to a minimum of _three_ nodes. When the Kubernets cluster size is met, after a node is added to the cluster, the join script will prompt to perform the migration. This migration is also available as a task within task.sh.  Please note the following requirements when choosing to have customers on both local and distributed storage.
```

File: src/markdown-pages/install-with-kurl/migrating-csi.md [hunk 2]
```diff
@@ -180,6 +180,43 @@ To migrate to a new CSI provisioner in a kURL cluster:
...
+
+1. Distributed storage requires a node count equal to or greater than 3.  This means that “n” can be 3 or larger.
+
+2. Automated local to distributed storage migrations require the following prerequisites:
+
+  - Rook 1.11.7 or Greater
```
Comment (rrpolanco) [inline src/markdown-pages/install-with-kurl/migrating-csi.md:191]: If this is true, the linter rule needs to change for this: https://github.com/replicatedhq/kurlkinds/pull/126/files#diff-ca0e9d53bb4c944ae7b52087e31382dfc05840f8fc0694b4d6cda36b273bd1e8R668

File: src/markdown-pages/install-with-kurl/migrating-csi.md [hunk 3]
```diff
@@ -180,6 +180,43 @@ To migrate to a new CSI provisioner in a kURL cluster:
...
+  - OpenEBS 3.7.0 or Greater
```
Comment (rrpolanco) [inline src/markdown-pages/install-with-kurl/migrating-csi.md:192]: If this is the version we're going with we'll need to update preflights and linter rules: https://github.com/replicatedhq/kurlkinds/pull/126/files#diff-ca0e9d53bb4c944ae7b52087e31382dfc05840f8fc0694b4d6cda36b273bd1e8R667

**Review by paigecalvert — APPROVED** [review]
Content looks great, just made some edits to reorganize a bit/reduce redundancy.

**Review by rrpolanco — APPROVED** [review]

**Pull request merged**